ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -371,6 +374,23 @@ def add_capacity(osd_size_capacity_requested):
     storageDeviceSets->count = (capacity reqested / osd capacity ) + existing count storageDeviceSets
 
     """
+    lvpresent = check_local_volume()
+    if lvpresent:
+        path = os.path.join(constants.EXTERNAL_DIR, "device-by-id-ocp")
+        utils.clone_repo(constants.OCP_QE_DEVICEPATH_REPO, path)
+        os.chdir(path)
+        utils.run_cmd("ansible-playbook devices_by_id.yml")
```

Comment:
please add a timeout here

Reply:
No need. Already has a timeout of 600s. More than enough.